ryan/ChristmasReee.java

```diff
@@ -0,0 +1,18 @@
+
+public class ChristmasReee {
+
+	public static void main(String[] args) {
+		System.out.println("          1          ");
+		System.out.println("         2 2         ");
+		System.out.println("        3   3        ");
+		System.out.println("       4     4       ");
+		System.out.println("      5       5      ");
+		System.out.println("       6     6       ");
+		System.out.println("      7       7      ");
+		System.out.println("     8         8     ");
+		System.out.println("    9           9    ");
+		System.out.println("   0             0   ");
+		System.out.println("    VAVAVAVAVAVAV    ");
+	}
```

Comment:
Excellent, it will work, but can I challenge you even further to using a loop to print the tree ? So not println for every line, but in a loop to print one line at a time?